src/views/includes/currencies.njk

```diff
@@ -0,0 +1,189 @@
+{% set CURRENCIES = [
+    { value: 'GBP', text: i18n.currency.GBP, selected: countryField === 'GBP'},
```

Comment:
i think we should use a different var name other than countryField here as it could clash with the one in countries.njk